### PR TITLE
CLDC-2967 Infer lar and irproduct values

### PR DESCRIPTION
**Pull request by kosiakkatrina**
- lar and irproduct values were not being set. Now we infer them based on rent_type
- add a task to update currently wrong lar values:
  - correct any logs that did not have lar values, but were supposed to
  - correct any logs that had wrong lar values
  - lar was being set to 2 (no) for all the imported logs that didn't have it set to 1 (yes). We only want lar to be 2 if it's affordable rent, in other cases we want it to be nil. Because of this being most of the logs, we will no be setting these to export.
- add a task to update currently wrong irproduct values

### CODE REVIEW

File: lib/tasks/recalculate_irproduct_values.rake [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  end
+  LettingsLog.exportable.where(rent_type: 3).where.not(irproduct: 1).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
+  LettingsLog.exportable.where(rent_type: 4).where.not(irproduct: 2).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
+  LettingsLog.exportable.where(rent_type: 5).where.not(irproduct: 3).each do |log| # irproduct was set wrong
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
```
Comment (kosiakkatrina) [inline lib/tasks/recalculate_irproduct_values.rake:14]: There are none of these on prod atm, but if rent_type is changed, these could happen

File: lib/tasks/recalculate_irproduct_values.rake [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  LettingsLog.exportable.where.not(rent_type: [3, 4, 5]).where.not(irproduct: nil).each do |log| # irproduct was set when it should have been nil
+    Rails.logger.info("Could not update irproduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
+  end
```
Comment (kosiakkatrina) [inline lib/tasks/recalculate_irproduct_values.rake:17]: Currently only 1 occurrence of this

File: lib/tasks/recalculate_irproduct_values.rake [hunk 3]
```diff
@@ -0,0 +1,18 @@
+desc "Forces to recalculate irproduct values"
+task recalculate_irproduct_values: :environment do
+  LettingsLog.exportable.where(rent_type: [3, 4, 5]).where(irproduct: nil).each do |log| # irproduct was never set
+    Rails.logger.info("Could not update irroduct for LettingsLog #{log.id}") unless log.update(values_updated_at: Time.zone.now)
```
Comment (davidm-m) [inline lib/tasks/recalculate_irproduct_values.rake]: Small type here in irproduct

**Review by davidm-m — APPROVED** [review]
One tiny typo but doesn't need a re-review, looks great!

File: lib/tasks/recalculate_irproduct_values.rake (hunk 3 above)
Comment (kosiakkatrina) [reply lib/tasks/recalculate_irproduct_values.rake]: Ah, good spot, thanks!

**Pull request merged**